docs/support.md

```diff
@@ -1,50 +1,61 @@
 # Supported systems, compatibility, troubleshooting & more
+
 ## Tested on...
+
 - Linux → Debian → Ubuntu → PopOS:
   - My main system, should work 100% fine
+- Windows 11
+  - [osfanbuff63](https://github.com/osfanbuff63)'s main system, should work fine
 
 ## Recommendations
+
 The program is optimized for Python 3.8 and above, as well as Linux and Windows.
 
 - `darkdetect`
-    - Used to check whether the OS theme is dark or not
+  - Used to check whether the OS theme is dark or not
     - Supports:
-        - macOS 10.14+
-        - Windows 10 1607+
-        - Linux with a dark GTK theme.
-    - Don't meed these requirements? Don't worry! The code will still work, just 
+      - macOS 10.14+
+      - Windows 10 1607+
+      - Linux with a dark GTK theme.
+    - Don't meed these requirements? Don't worry! The code will still work, just dark mode will not be detected.
 
 ## Troubleshooting
+
 ### Pixelated font / issues with Anaconda (Linux)
+
 Does the font look pixelated and emojis are not loading?
 
-![](media/conda-bug.png)
+![Font looking pixelated due to Anaconda](media/conda-bug.png)
 
-Don't worry! Try the following commands (AFTER trying [install/linux.sh](install/linux.sh) and [install/linux.sh](install/linux.sh)):
+Don't worry! Try the following commands (AFTER trying [install/linux.sh](install/linux.sh)):
 
-```
+```bash
```

Comment:
Shouldn't this also have `#!/bin/bash` at beginning of script?

Reply:
Probably yeah